release.mk

```diff
@@ -818,15 +817,9 @@ endif
 $O/.stamp_ide_linux_eclipse: $O/$(ECLIPSE_VANILLA_LINUX_TGZ) $O/$(IDE_PLUGINS_ZIP)
 	mkdir -m775 -p $O/$(IDE_LINUX_INSTALL)
 	tar xf $< -C $O/$(IDE_LINUX_INSTALL)
-	$O/$(IDE_LINUX_INSTALL)/eclipse/eclipse \
-	    -application org.eclipse.equinox.p2.director \
-	    -noSplash \
-	    -repository $(ECLIPSE_REPO),jar:file:$(realpath $O/$(IDE_PLUGINS_ZIP))\!/ \
-	    -installIU $(ECLIPSE_PREREQ),com.arc.cdt.feature.feature.group
-	# Eclipse will create a bunch of repos with local paths, that will not
-	# work for end-users, hence those repos must be manually removed.
-	sed -i -e "/$(subst /,_,$O)/ d" \
-	    $O/$(IDE_LINUX_INSTALL)/eclipse/p2/org.eclipse.equinox.p2.engine/profileRegistry/epp.package.cpp.profile/.data/.settings/org.eclipse.equinox.p2.*
+	unzip $O/${IDE_PLUGINS_ZIP} -d $O/$(IDE_LINUX_INSTALL)/eclipse/dropins
+	rm $O/$(IDE_LINUX_INSTALL)/eclipse/dropins/artifacts.jar
```

Comment:
Should be `rm -f` instead of `rm` to avoid any potential issues with nonexistent files, etc.

Reply:
done